hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/services/ITestAbfsClient.java

```diff
@@ -685,4 +692,69 @@ public void testExpectHundredContinue() throws Exception {
             .describedAs("The expect header is not false")
             .isFalse();
   }
+
+  @Test
+  public void testTimerNotInitialize() throws Exception {
+    // Create an AzureBlobFileSystem instance.
+    final Configuration configuration = getRawConfiguration();
+    AbfsConfiguration abfsConfiguration = new AbfsConfiguration(configuration, ACCOUNT_NAME);
+
+    AbfsCounters abfsCounters = Mockito.spy(new AbfsCountersImpl(new URI("abcd")));
+    AbfsClientContext abfsClientContext = new AbfsClientContextBuilder().withAbfsCounters(abfsCounters).build();
+
+    // Get an instance of AbfsClient.
+    AbfsClient client = new AbfsDfsClient(new URL("https://azure.com"),
```

Comment:
abstract class instantiated, there are methods such as getMockAbfsClient and others in ITestAbfsClient class that can help to get an object